packages/ozone/src/util.ts

```diff
@@ -0,0 +1,26 @@
+import { AxiosError } from 'axios'
```

Comment:
untanglining these dependencies is a big enough pain that i just copied this over

I'd really like a generic place for "service utils"